src/pages/[platform]/build-a-backend/troubleshooting/library-not-configured/index.mdx

```diff
@@ -0,0 +1,149 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Library Not Configured',
```

Comment:
```suggestion
  title: 'Troubleshoot "Library Not Configured" errors',
```

Reply:
I updated this to 'Troubleshoot configuration errors' since LibaryNotConfigured is not the actual error message sent.